services/bettersocial/index.js

```diff
@@ -39,6 +39,7 @@ const BetterSocialCore = {
     sendChatNotification: require('./fcmToken/sendChatNotification')
   },
   post: {
+    blockAnonymousComment: require('./post/blockAnonymousComment'),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `blockAnonymousComment` is added correctly to the `post` module in `BetterSocialCore`. However, based on the PR title and summary of changes, it seems like the function should be named `blockAnonymousCommentV2`, not `blockAnonymousComment`. Please ensure that the correct function is being referenced.
```diff
-    blockAnonymousComment: require('./post/blockAnonymousComment'),
+    blockAnonymousCommentV2: require('./post/blockAnonymousCommentV2'),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->